tests/unit/test_metrics.py

```diff
@@ -0,0 +1,45 @@
+# Copyright Splunk Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from unittest import TestCase, mock
+
+from opentelemetry.sdk.metrics import MeterProvider
+
+from splunk_otel.metrics import _configure_metrics, start_metrics
+from splunk_otel.version import __version__
+
+# pylint: disable=protected-access
+
+
+class TestMetrics(TestCase):
```

Comment:
Should there be a test to check whether the global meter provider has been set? (that `get_meter_provider` returns the global meter provider)

Reply:
Added a test to check if the global meter provider is not a no-op meter provider that is returned by default if nothing is set and in fact is an instance of MeterProvider.